content/md/products-super-power/improve-data-quality.md

```diff
@@ -59,37 +59,41 @@ To help you improve your products **enrichment** and **consistency**, simply fol
 From this panel, you will know which attributes need to be improved and how to improve them.
 
 ::: info
-Links can take you either to a specific attribute or to the list of improvable attributes in the attribute panel.
+From the 'data quality panel', links on attribute names can take you to a specific attribute. By clicking on the 'Display all improvable attributes', you can easily access a product edit form filtered on all the attributes that need to be checked for consistency.
 :::
 
-
-![Image data quality variant panel](../img/data-quality-variant-panel.png)
-
-For variant products, the data quality panel lists all the attributes that need improvements to reach a higher data quality grade, whether at model or variant level.
-
-::: info
-Black attribute labels are not clickable and must be improved at product level.
-Purple attribute labels are clickable and can be improved at the variant level.
-:::
-
-
 To help you even further, in the `Attributes` panel, you can select spelling corrections and title formatting improvements.
 
 ![Image data quality spellcheck corrections](../img/data-quality-spelling.png)
 
 ::: info
-Even if they don't have permission to edit, all PIM users will be able to read data quality insights on all attributes.
+Spell check is available for the English, French, German, Spanish, Italian, Dutch, Danish, Swedish, Norwegian and Portuguese (Brazilian) locales.
+Spell check is usable with localisable text and text area attributes.
 :::
 
 ::: info
-Spell check is available for the English, French, German, Spanish, Italian, Dutch, Danish, and Swedish locales.
-Spell check is usable with text and simple text area attributes.
+Title formatting suggestion is available for English locales.
 :::
 
 ::: info
-Title formatting suggestion is available for English locales.
+Even if they don't have permission to edit, all PIM users will be able to read data quality insights on all attributes.
 :::
 
-::: tips
-From the `data quality insights` panel, by clicking on the "Display all improvable attributes", you can easily access a product edit form filtered on all the attributes that need to be checked for consistency.
-:::
+# Improve data quality on variant products
+
+If you work with variant products, you will see slight differences between grades and recommendations for simple, product models and variant products.
```

Comment:
I feel there is a word missing here:  for simple products?

Reply:
yes, there is. thanks